src/internal/modules/billing/services/transactions-csv.js

```diff
@@ -46,19 +46,6 @@ const getInvoiceAccountData = invoiceAccount => {
   };
 };
 
-const getInvoiceData = invoice => ({
```

Comment:
These properties appear to be coming from an `invoice` model not an `invoiceLicence`, so wondering if they need to be deleted?

Did it break without these deletions?

Reply:
I think that while the data was mapped against an invoice model (or an invoice-like model), the data itself seemed to come from `licence_holder_address`.

Or so I assumed, because prior to removing this, the UI responded with 'An error occurred' page.